### DISCUSSION TITLE
Can't login following registration

### DISCUSSION THREAD

**Discussion by DamianPike** [datalust/seq-tickets#1400] [Help] [answered]
Hi. A newbie situation here - any help welcome.

 I've downloaded Seq, run through the Service Administration and entered a password.

However, on attempt to log in, I get "Could not log in. Invalid username or password".

I have downloaded the CLI - attempted to stop application (seq stop) prior to attempt to reset authorisation, however I get 'Access is denied'.

I have tried uninstalling and reinstalling, but it's using the configuration file - so no joy. 

This is for a single user, development machine, use - but do I need some sort of licence api key?

Thanks

**Comment by nblumhardt** [comment, marked as answer]
Hi Damian,

Thanks for your message. If you're not concerned about saving any existing configuration info, you can open an administrative command prompt, and run:

```
seq service stop
seq service uninstall
rm C:\ProgramData\Seq -Recurse -Force
```

and then run through Seq Service Administration (from the start menu) again.

Let me know if this does the trick!

Best regards,
Nick

**Reply by DamianPike** [reply]
Hi - thanks for quick response. Alas, this doesn't work - I get 
"Stopping Seq... 
Cannot open 'Seq' Service on computer '.
Access is denied."

**Reply by DamianPike** [reply]
Resolved!

**Reply by nblumhardt** [reply]
Fantastic 👍
